pinot-controller/src/main/resources/app/pages/TenantDetails.tsx

```diff
@@ -445,6 +450,17 @@ const TenantPageDetails = ({ match }: RouteComponentProps<Props>) => {
     }
   };
 
+  const handleRebalanceTableStatus = async () => {
+    try{
+      setShowRebalanceServerStatus(true);
+      const rebalanceServerJobs = await PinotMethodUtils.fetchTableJobs(tableName, "TABLE_REBALANCE");
+      setRebalanceTableJobsData(rebalanceServerJobs as RebalanceTableSegmentJobs);
+    } catch(error) {
+      dispatch({type: 'error', message: error, show: true});
+      setShowRebalanceServerStatus(false);
+    }
+  };
+
```

Comment:
Are we showing loading until api fetch is complete?
user clicks on rebalance status button -> open dialog -> call api -> api fails -> close dialog ??

what if api takes 3 seconds? what about loading indicator ?

Reply:
IMO we should make api call in the Dialog itself in useEffect

Reply:
Moved API call to dialog!